api/api-facade/api-git/pom.xml

```diff
@@ -44,9 +44,15 @@
         </dependency>
         <dependency>
             <groupId>org.eclipse.dirigible</groupId>
-            <artifactId>dirigible-core</artifactId>
+            <artifactId>dirigible-core-git</artifactId>
             <version>7.0.0-SNAPSHOT</version>
         </dependency>
+        <dependency>
+            <groupId>org.eclipse.dirigible</groupId>
+            <artifactId>dirigible-commons-test</artifactId>
+            <version>6.1.11</version>
```

Comment:
Is there a specific reason to reference version **6.1.11** instead of **7.0.0-SNAPSHOT**. Pointing to a specific version could lead in unwanted behaviour after some time and few version increases.